utils.py

```diff
@@ -97,3 +99,92 @@ def get_details_from_id(alert_id):
 
     return retVal
 
+def parse_details_to_file_bug(details, oldest_alert):
+    branch = details['branch']
+    test = details['test']
+    platform = details['platform']
+    percent = details['percent']
+    push_date = details['push_date']
+    summary = ''
+
+    #add regression values
+    min_percent = min(percent)
+    max_percent = max(percent)
+    min_percent = re.findall('(?P<number>\d+)', min_percent)[0]
+    max_percent = re.findall('(?P<number>\d+)', max_percent)[0]
+    summary = summary + '%s-%s' %(min_percent, max_percent) + '% '
+
+    #add platform
+    flag_linux = 0
+    flag_win = 0
+    flag_mac = 0
+    flag_android = 0
+    for p in platform:
+        if 'Ubuntu' in p:
+            flag_linux = 1
+        if 'WIN' in p:
+            flag_win = 1
+        if 'Mac' in p:
+            flag_mac = 1
+        if 'Android' in p:
+            flag_android = 1
+    add = ''
+    if flag_linux:
+        add = add + 'Linux*/'
+    if flag_win:
+        add = add + 'Win*/'
+    if flag_mac:
+        add = add + 'MacOS*/'
+    if flag_android:
+        add = add + 'Android 4.0 Tegra/'
+
+    add = add[:-1]+' '
+    summary = summary + add
+
+    #add Test
+    add = ''
+    for t in sorted(set(test)):
+        add = add + t+'/'
+    add = add[:-1]+' '
+
+    summary = summary + add
+
+    summary = summary + 'regression on '
+
+    #add branch
+    add = oldest_alert[0]
+    add = add+' '
+
+    summary = summary + add
+
+    summary = summary + 'on '
+
+    #add date
+    summary = summary + oldest_alert[4].strftime("%B %d, %Y") + ' from push %s' %details['keyrev']
+
+    #Creating Description
+    desc = """We have verified a regression, here is a list to all the regressions and improvements that we know of so far:
+    %s/alerts.html?rev=%s&showAll=1&testIndex=0&platIndex=0
+    Viewing the above alerts in alert manager you can view each alert as well as a link to the graph 
+    showing the regression. There is also a link to treeherder showing the jobs in a pushlog format.
+    To learn more about this test, please visit: %s
+    to run this locally, follow these instructions to get setup:
+    https://wiki.mozilla.org/Buildbot/Talos/Running#Running_locally_-_Source_Code
+    and then run:
+    talos --develop -e <path>/firefox -a %s
+
+    If you want to push to try, you can use this syntax:
+    try: -b o -p linux -u none -t %s
```

Comment:
here we hardcode linux, most people will see the right thing to do, but we could be smarter here.  Here is the list of platforms to choose from: try: -b o -p linux,linux64,macosx64,win32,win64,android-api-11 -u none -t none, that should map into the platform map you find for the subject line :)  for reference those platforms are determined by trychooser:
http://trychooser.pub.build.mozilla.org/